ui/app/AppLayouts/Wallet/views/walletconnect/sdk/src/index.js

```diff
@@ -72,6 +72,14 @@ window.wc = {
         });
     },
 
+    getPairings: function () {
+        return window.wc.core.pairing.getPairings();
+    },
+
+    disconnect: function (topic) {
+        return window.wc.core.pairing.disconnect({ topic: topic});
+    },
+
     registerForSessionRequest: function (callback) {
         window.wc.web3wallet.on("session_request", callback);
```

Comment:
@stefandunca is it safe to call many many times this? Like does web3wallet handle multiple listeners for the same event with or without overwriting?

Reply:
`registerForSessionRequest` is called only once after SDK is initialized. Will improve though, with my current work.